firmware/library/L1_Drivers/pwm.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <cstdint>
+#include <cstdio>
+#include "L0_LowLevel/LPC40xx.h"
+#include "L1_Drivers/pin_configure.hpp"
```

Comment:
cstdio, pin_configure, and LPC40xxx are not used here, they shouldn't be included.